README.md

```diff
@@ -452,15 +452,27 @@ All statements above will return false if used with `===`
 
 <br/><br/>
 
-## ![✔] 4.3 Detect code issues with a linter
+## ![✔] 4.3 Structure tests by the AAA pattern
+
+**TL;DR:** Structure your tests with 3 well-separated sections: Arrange, Act & Assert (AAA). The first part includes the test setup, then the execution of the unit under test and finally the assertion phase. Following this structure guarantees that the reader spends no brain CPU on understanding the test plan
+
+
```

Comment:
There is a single line for the common consistency.

Reply:
@oshliaer good catch!